app/models/review.rb

```diff
@@ -1,5 +1,5 @@
 class Review < ApplicationRecord
   belongs_to :user
   belongs_to :shelter
-  validates_presence_of :title, :rating, :content, :name_of_user
+  validates_presence_of :title, :rating, :content, :name_of_user, :user_id
 end
```

Comment:
Good job at adding this to our validations that is an important part to validate considering a review belongs to a user